test/Tests/src/System/File_Spec.enso

```diff
@@ -359,6 +528,22 @@ spec =
                 n.read_text . should_equal "new content"
             [f, bak, n0, n1, n2, n4].each .delete
 
+        Test.specify "should create a backup when writing a dry run file with Context.Output disabled" <|
```

Comment:
```suggestion
        Test.specify "should not create a backup when writing a dry run file with Context.Output disabled" <|
```

The description of the test and the actual contents of the test don't seem to match, please make it consistent as that is quite confusing.